proto/sequencerblockapis/astria/sequencerblock/v1alpha1/optimistic_block.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package astria.auctioneer.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+import "astria/execution/v1alpha2/execution.proto";
+import "astria/sequencerblock/v1alpha1/block.proto";
+import "astria/primitive/v1/types.proto";
+
+// The "BaseBlock" is the information needed to simulate bundles on top of
+// a Sequencer block which may not have been committed yet.
+message BaseBlock {
+    // This is the block hash for the proposed block.
+    bytes sequencer_block_hash = 1;
+    // The hash of previous rollup block, which new block will be created on top of.
+    bytes prev_rollup_block_hash = 3;
```

Comment:
Can we assume that this will be the latest soft block of the rollup? Because I believe we would have to query the soft block from auctioneer via the `GetCommitmentState` method which might add to latency.

Reply:
we can probably remove this? what is it used for